src/main/java/org/aksw/simba/squirrel/data/uri/filter/KnownUriFilter.java

```diff
@@ -29,6 +29,20 @@
      */
     public void add(CrawleableUri uri, long lastCrawlTimestamp, long nextCrawlTimestamp);
 
+    /**
+     * Add the given hash values for the given uris. Hash values are contained in the uris.
+     *
+     * @param uris       The given uris.
+     */
+    public void addHashValuesForUris(List<CrawleableUri> uris);
+
+
+    /**
+     * Get all uris.
+     * @return All uris contained in this filter.
+     */
+    public List<CrawleableUri> getAllUris();
```

Comment:
That is one of the methods for which we can already predict that it might simply break in the future because of the amount of URIs :disappointed: